### ISSUE TITLE
send clipboard

### ISSUE THREAD

**Issue by goropikari** [korsbo/Latexify.jl#5]
By using [`clipboard`](https://docs.julialang.org/en/stable/stdlib/base/#Base.clipboard-Tuple{Any}), we can send a LaTeX command to clipboard. So we can omit the process of printing a LaTeX command and copying it.

For example,
```julia
return LaTeXString(str)
```
&darr;
```julia
latexstr = LaTeXString(str)
clipboard(latexstr)
return latexstr
```
Would you implement above feature?

**Comment by korsbo** [comment]
This is a cool idea (you're full of them!). I am intrigued but a bit afraid of messing with peoples clipboard. Given that every user will not know the package intimately and could be unaware of such a feature, is it acceptable to allow a program to silently alter something outside of the program itself?

The implementation would be dead-easy, so the only question is whether this is a desirable feature.

**Comment by korsbo** [comment]
We could of course let this behaviour be governed by some global variable and provide a function for toggling it. 

```julia
latexstr = LaTeXString(str)
COPY_TO_CLIPBOARD && clipboard(latexstr)
return latexstr
```

```julia 
module Latexify
export copy_to_clipboard

const COPY_TO_CLIPBOARD = false
function copy_to_clipboard( x::Bool ) 
    COPY_TO_CLIPBOARD = x
end
```

**Comment by goropikari** [comment]
I agree that we control the behavior by the global variable.
But because the scope of functions is [Hard Local Scope](https://docs.julialang.org/en/stable/manual/variables-and-scoping/#Hard-Local-Scope-1), you should define a variable and a function as follows.
```julia
COPY_TO_CLIPBOARD = false
function copy_to_clipboard(x::Bool)
    global COPY_TO_CLIPBOARD = x
end

foo() = COPY_TO_CLIPBOARD && println("Hello World!")
```
Output
```julia
julia> foo()
false

julia> copy_to_clipboard(true)
true

julia> foo()
Hello World!

julia> copy_to_clipboard(false)
false

julia> foo()
false
```


On the other hand,
```julia
const COPY_TO_CLIPBOARD = false
function copy_to_clipboard( x::Bool ) 
    COPY_TO_CLIPBOARD = x
end

foo() = COPY_TO_CLIPBOARD && println("Hello World!")
```
Output
```julia
julia> foo()
false

julia> copy_to_clipboard(true)
true

julia> foo()
false
```

**Comment by korsbo** [comment]
I have implemented this in the master branch. I will wait with tagging a release until I have had time to update the docs and write a test. 

Thanks for the suggestion, this is a nice improvement!

**Comment by korsbo** [comment]
And also, thanks for educating me on Julia programming. It's much appreciated!

**Comment by goropikari** [comment]
Thanks for accepting my requests!
I also  learned a lot. Thank you:smile:

**Comment by korsbo** [comment]
tagged and released as v0.1.1
